index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Music Festival</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="main-nav bg-secondary">
+        <div class="container">
+            <ul class="main-nav-ul float-right">
+                <li class="main-nav-i"><i class="fa-brands fa-facebook-f"></i></li>
+                <li class="main-nav-i"><i class="fa-brands fa-twitter"></i></li>
+                <li class="main-nav-i">English</li>
+                <li class="main-nav-i">My Page</li>
+                <li class="main-nav-i">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <nav class="second-nav fixed-top-nav" id="first-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-image"></i>
+            <div class="navigation">
+                <img class="navigation-logo" src="assets/musicFestivalLogo.png" width="75px" alt="Festival">
+                <ul class="second-nav-ul float-right">
+                    <li><a class="second-nav-i" href="about.html">About</a></li><br>
+                    <li><a class="second-nav-i" href="#program">Program</a></li><br>
+                    <li><a class="second-nav-i" href="#join">Join</a></li><br>
+                    <li><a class="second-nav-i" href="#sponsor">Sponsor</a></li><br>
+                    <li><a class="second-nav-i" href="#updates">News</a></li><br>
+                    <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+                </ul>
```

Comment:
- [ ] As suggested by the previous code reviewer, the main nav background color needs to be white as it is in the Behance design. Also `cc Campaign` button is overflowing into the top menu as you see below :point_down: so adjust the styling of it and make sure its margins top and down match the design

- [ ] Also while you were making the changes, Your logo is overflowing to the bottom so ensure that you fix it. Kindly make sure your top nav background color looks like the Behance design `#272A31`:pray: 
  

- [ OPTIONAL ] Ensure that the `cc campaign` button border color matches the design :+1:  

- [ OPTIONAL ] Please your text font does not match the design as you can see. and make sure that the `About` link color match the design

  ### Your design
  ![image](https://github.com/Kgomotso196/FirstCapstoneProject/assets/92303891/00847a92-bf51-4a99-b029-688c91cbc7bd)
  
  ### Behance design
  ![image](https://github.com/Kgomotso196/FirstCapstoneProject/assets/92303891/b58b0b2e-0677-42da-a807-c4481af0fa90)